### PR TITLE
Adding FIO multithroughput

### PR DESCRIPTION
**Pull request by deep1712**

### CODE REVIEW

File: website/docs/workloads/fio/fio-profiles.md [hunk 1]
```diff
@@ -269,6 +269,20 @@ This profile uses an algorithm to determine the total number of jobs/threads as
   ./VirtualClient --profile=PERF-IO-FIO-DISCOVERY.json --system=Demo --timeout=1440 --packageStore="{BlobConnectionString|SAS Uri}"  --parameters="QueueDepths="4,16,256"
   ```
 
+-----------------------------------------------------------------------
+
+### PERF-IO-FIO-MULTITHROUGHPUT.json
+
```
Comment (yangpanMS) [inline website/docs/workloads/fio/fio-profiles.md:275]: There is already a PERF-IO-FIO-MULTITHROUGHPUT.json section below, could you merge the two? And also please give more concrete examples on the commands running on the random io disks and sequential io disks, concurrently or not concurrently

File: src/VirtualClient/TestResources/expectedoltp-c.fio2.jobfile [hunk 2]
```diff
@@ -18,6 +18,7 @@ time_based=1
 [randomreader]
 new_group
 bs=8K
```
Comment (yangpanMS) [inline src/VirtualClient/TestResources/expectedoltp-c.fio2.jobfile:20]: Looks like you changed both fio1 and fio2 to assume 2 disks. Could you keep the original 1 disk test and then add a 2 or even 8 disks test?

**Review by yangpanMS — COMMENTED** [review]
It looks good, I just need to understand more about how the random and sequential disks are selected, and their count when there are 1/2/16 disks on system.

**Review by yangpanMS — APPROVED** [review]
I'm approving this, but it's still not very clear in the doc why we are treating sequential and random IO differently. It would be good to explain that.

**Comment by yangpanMS** [comment]
Could you pull the latest again, I will merge for you if it builds

File: src/VirtualClient/VirtualClient.Main/profiles/PERF-IO-FIO-MULTITHROUGHPUT.json [hunk 3]
```diff
@@ -12,7 +13,8 @@
         "TargetIOPS": "5000",
         "TargetPercents": "10,40,90,98,100,102,110",
         "DirectIO": true,
-        "InitializeDisksInParallel": true
+        "InitializeDisksInParallel": true,
+        "SequentialDiskCount": "1"
```
Comment (brdeyo) [inline src/VirtualClient/VirtualClient.Main/profiles/PERF-IO-FIO-MULTITHROUGHPUT.json:17]: SequentialDiskCount: Let's make the default behavior to leave this parameter as null/undefined. In the default scenario where the user does not define this, we apply random I/O to half of the disks and sequential I/O to the other half (e.g. if there are 4 disks, we apply random I/O to 2 and sequential I/O to 2). For cases where we have an odd number of disks, we should apply random I/O to just over half and sequential I/O to the rest (e.g. if there are 5 disks, we apply random I/O to 3 disks and sequential I/O to 2).

File: src/VirtualClient/VirtualClient.Main/profiles/PERF-IO-FIO-MULTITHROUGHPUT.json [hunk 4]
```diff
@@ -61,7 +65,8 @@
                 "TargetIOPS": "$.Parameters.TargetIOPS",
                 "TargetPercents": "$.Parameters.TargetPercents",
                 "TemplateJobFile": "oltp-c.fio.jobfile",
-                "Tags": "IO,FIO,Multi-Throughput,OLTP"
+                "Tags": "IO,FIO,Multi-Throughput,OLTP",
+                "SequentialDiskCount": "$.Parameters.SequentialDiskCount"
```
Comment (brdeyo) [inline src/VirtualClient/VirtualClient.Main/profiles/PERF-IO-FIO-MULTITHROUGHPUT.json:69]: Keep the 'SequentialDiskCount' parameter with the other Sequential* parameters.

File: src/VirtualClient/VirtualClient.Main/profiles/PERF-IO-FIO-MULTITHROUGHPUT.json (hunk 3 above)
Comment (deep1712) [reply src/VirtualClient/VirtualClient.Main/profiles/PERF-IO-FIO-MULTITHROUGHPUT.json:17]: I discussed this with Srivanthi
The Default case for OLTP is using 1 log disk(Sequential) & others as data disks(Random).

**Review by brdeyo — APPROVED** [review]

**Pull request merged**